src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    public Truck() {
+    }
+
+    @Override

```

Comment:
remove constructor - no reason for it
```suggestion
public class Truck extends Machine {
    @Override
```